### PR TITLE
[glib]Fix missing dependency selinux.

### PR DESCRIPTION
**Pull request by JackBoosY**
Related: #7978.

### CODE REVIEW

File: ports/glib/portfile.cmake [hunk 1]
```diff
@@ -10,6 +10,10 @@ if (NOT VCPKG_CMAKE_SYSTEM_NAME)
     vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY ONLY_DYNAMIC_CRT)
 endif()
 
+if (NOT VCPKG_TARGET_IS_WINDOWS AND NOT EXISTS "/usr/include/selinux")
+    message(FATAL_ERROR "selinux must be installed before glib can build. Install them with \"apt-get install selinux\".")
```
Comment (dan-shaw) [inline ports/glib/portfile.cmake]: This should only be a warning - as the user does not have to use the system version of selinux . If `/usr/include/selinux` exists, we should not print a message.

Comment (JackBoosY) [reply ports/glib/portfile.cmake]: @dan-shaw But I found glib cannot be built without selinux support. Can you confirm this?

Comment (cbezault) [reply ports/glib/portfile.cmake]: His point is that selinux can be installed in any number of places and that will still work.

Comment (cbezault) [reply ports/glib/portfile.cmake]: You can also disable selinux with `-Dselinux=disabled`.

**Comment by grdowns** [comment]
Thanks @JackBoosY!

**Pull request merged**